code/noRAM/app/src/main/java/com/example/noram/OrganizerCreateEventFragment.java

```diff
@@ -40,10 +60,25 @@ public class OrganizerCreateEventFragment extends Fragment implements DatePicker
     int endMinute;
     private LocalDateTime startDateTime;
     private LocalDateTime endDateTime;
-    AppCompatButton editStartDateTime;
-    AppCompatButton editEndDateTime;
+    private AppCompatButton editStartDateTime;
+    private AppCompatButton editEndDateTime;
     View createdView;
 
+    private Uri imageUri;
+
+    AppCompatButton editStartTime;
+    private AppCompatButton editEndTime;
```

Comment:
are editStartDateTime and editStartTime different buttons? they sound like they're duplicates, but i might be wrong

Reply:
I forgot to remove these after the merge. we use private AppCompatButton editStartDateTime;
    private AppCompatButton editEndDateTime;